src/v/cloud_storage/partition_manifest.cc

```diff
@@ -248,7 +255,9 @@ partition_manifest::partition_manifest(
 // backend and other S3 API implementations might benefit from that.
 
 remote_manifest_path generate_partition_manifest_path(
-  const model::ntp& ntp, model::initial_revision_id rev) {
+  const model::ntp& ntp,
+  model::initial_revision_id rev,
+  std::string_view suffix) {
```

Comment:
Maybe use an enum instead of an arbitrary string here, as we only expect two possible values.

Reply:
Yes, reused manifest_format